packages/lab/src/combo-box-next/useComboBox.tsx

```diff
@@ -64,25 +63,23 @@ export const useComboBox = ({
   }, [selectedItem]);
 
   const focusHandler = (event: FocusEvent<HTMLInputElement>) => {
-    setOpen(true);
```

Comment:
This deserves a changeset mention.

Reply:
This line is in the changeset:
Combobox: keyboard navigation modified so Tab sets focus on the component and Enter opens the List, aligns to Dropdown - Do i need to be more specific ?

FloatingUI handles the open of the floating element on click of the trigger so ```setOpen(true)``` isn't explicitly needed in the focus handler